cmd/harmony/config/config.go

```diff
@@ -7,53 +7,48 @@ import (
 	"os"
 	"strings"
 
-	"github.com/harmony-one/harmony/internal/cli"
 	nodeconfig "github.com/harmony-one/harmony/internal/configs/node"
 	"github.com/pelletier/go-toml"
-	"github.com/spf13/cobra"
 )
 
-// harmonyConfig contains all the configs user can set for running harmony binary. Served as the bridge
+// HarmonyConfig contains all the configs user can set for running harmony binary. Served as the bridge
```

Comment:
actually, I think that it is better to have the latest config always to be named the same (e.g. HarmonyConfig, not HarmonyConfigV2). this will keep the fact that there might be several config versions private to the config package, and also keep the possible code changes for migrating to new config versions (v3, v4 etc) to the single package.

Reply:
Yea, I totally agree with you about abstracting all details about config loading. What I have in mind is:

1. Latest config should always be named as `HarmonyConfig`
2. V2 should be named privately like `harmonyConfigV2`

Something like:

```
// config.go
type HarmonyConfig harmonyConfigV2

// v2.go
type harmonyConfigV2 struct {...}
```

Thus the two points you give in the comment will be still well kept:

1. Different versions are named privately.
2. Latest version is always called as `HarmonyConfig`

Moreover, his code will more explicitly show that the latest HarmonyConfig is of version V2, and will have smaller code changes when updating to a new version (Only need to redefine `type HarmonyConfig harmonyConfigV3` instead of rename the file to V2). 

Or do you have any other concerns about this? We can have a more in depth discussion.